src/incidencematrix.jl

```diff
@@ -78,6 +92,27 @@ Base.@propagate_inbounds function col(M::IncidenceMatrix, j::Base.Integer)
     return to_one_based_indexing(_col(M, convert(Int64, j)))
 end
 
+function _findnz(M::IncidenceMatrix)
+    len = sum(([length(row(M, i)) for i in 1:size(M, 1)]))
```

Comment:
a vector of lengths is allocated here as well. `sum(length, (row(M, i) for i in 1:size(M, 1)))` should do.

Also: hopefully `row(M, i)` does not allocate. The generator here is usually written as `eachrow(M)`, but we probably don't wrap it, do we?